go.mod

```diff
@@ -167,16 +168,16 @@ require (
 	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5
 	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
 	golang.org/x/mod v0.5.1
-	golang.org/x/net v0.0.0-20211020060615-d418f374d309
-	golang.org/x/oauth2 v0.0.0-20211005180243-6b3c2da341f1
+	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
+	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	golang.org/x/sys v0.0.0-20211102192858-4dd72447c267
+	golang.org/x/sys v0.0.0-20220128215802-99c3d69c2c27
 	golang.org/x/text v0.3.7
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
 	golang.org/x/tools v0.1.7
-	google.golang.org/api v0.58.0
-	google.golang.org/genproto v0.0.0-20211021150943-2b146023228c
-	google.golang.org/grpc v1.41.0
+	google.golang.org/api v0.65.0
```

Comment:
Is this update needed? I would prefer not to update the dependency **and** adding a feature in the same PR.

Reply:
yes, `cloud.google.com/go/compute  v1.0.0` require  `google.golang.org/api v0.65.0`.  

I agree with you, feature and dependency should be committed in different PR, then I have to use package `google.golang.org/api ` to resolve this issue and replace this logic with new package in the future.   Will it be a bit complicated?

In my  opinion, `google.golang.org/grpc` and `google.golang.org/api` have  minor version updates and there is no breaking change, it may be safe to upgrade a little minor version number.

https://github.com/googleapis/google-cloud-go/blob/compute/v1.1.0/compute/go.mod

Reply:
Ok, so let's go with the overall change and we will spend some more time testing this to make sure everything is correct.

Reply:
I have reconsidered your suggestion，it is better not to make changes to dependency, that would be more clear and simple for testing and reviewing. I will rewrite that logic in a few days. Thanks for your help.